README.md

```diff
@@ -95,235 +95,226 @@ To configure the package behaviour, you can modify your `network_params.json` fi
+  #  Valid values are nimbus, lighthouse, lodestar, teku, and prysm
+  cl_client_type: lighthouse
+
+  #  The Docker image that should be used for the EL client; leave blank to use the default for the client type
+  #  Defaults by client (note that Prysm is different in that it requires two images - a Beacon and a validator - separated by a comma):
+  #  - lighthouse: sigp/lighthouse:latest
+  #  - teku: consensys/teku:latest
+  #  - nimbus: statusim/nimbus-eth2:multiarch-latest
+  #  - prysm: gcr.io/prysmaticlabs/prysm/beacon-chain:latest,gcr.io/prysmaticlabs/prysm/validator:latest
+  #  - lodestar: chainsafe/lodestar:next
+  cl_client_image: ""
+
+  #  The log level string that this participant's EL client should log at
+  #  If this is emptystring then the global `logLevel` parameter's value will be translated into a string appropriate for the client (e.g. if
+  #   global `logLevel` = `info` then Teku would receive `INFO`, Prysm would receive `info`, etc.)
+  #  If this is not emptystring, then this value will override the global `logLevel` setting to allow for fine-grained control
+  #   over a specific participant's logging
+  cl_client_log_level: ""
+
+  #  A list of optional extra params that will be passed to the CL client Beacon container for modifying its behaviour
+  #  If the client combines the Beacon & validator nodes (e.g. Teku, Nimbus), then this list will be passed to the combined Beacon-validator node
+  beacon_extra_params: []
+
+  #  A list of optional extra params that will be passed to the CL client validator container for modifying its behaviour
+  #  If the client combines the Beacon & validator nodes (e.g. Teku, Nimbus), then this list will also be passed to the combined Beacon-validator node
+  validator_extra_params: []
+
+  # A set of parameters the node needs to reach an external block building network
+  # If `null` then the builder infrastructure will not be instantiated
+  # Example:
+  #
+  # "relay_endpoints": [
+  #   "https:#0xdeadbeefcafa@relay.example.com",
+  #   "https:#0xdeadbeefcafb@relay.example.com",
+  #   "https:#0xdeadbeefcafc@relay.example.com",
+  #   "https:#0xdeadbeefcafd@relay.example.com"
+  #  ]
+  builder_network_params: null
+
+  # Resource management for el/beacon/validator containers
+  # CPU is milicores
+  # RAM is in MB
+  # Defaults are set per client
+  el_min_cpu: ''
+  el_max_cpu: ''
+  el_min_mem: ''
+  el_max_mem: ''
+  bn_min_cpu: ''
+  bn_max_cpu: ''
+  bn_min_mem: ''
+  bn_max_mem: ''
+  v_min_cpu: ''
+  v_max_cpu: ''
+  v_min_mem: ''
+  v_max_mem: ''
+
+  # Snooper can be enabled with the `snooper_enabled` flag per client or globally
+  # Defaults to false
+  snooper_enabled: false
+
+  # Count of nodes to spin up for this participant
+  # Default to 1
+  count: 1
+
+  # Count of the number of validators you want to run for a given participant
+  # Default to null, which means that the number of validators will be using the network parameter num_validator_keys_per_node
```

Comment:
indentation

Reply:
?

Reply:
i was wondering if he `network parameter` can be on a separate line so that its justiified with the # Default

Reply:
network parameter is a few lines below this tho. I still don't quite get what you want me to change.

Reply:
the default isn't in the same vertical line as 'parameter'
<img width="800" alt="Screenshot 2023-10-25 at 1 03 35 PM" src="https://github.com/kurtosis-tech/ethereum-package/assets/6670312/a500c7ef-baf5-41b7-a981-1cca26cef5b2">
```suggestion
  # Default to null, which means that the number of validators will be using the
  # network parameter num_validator_keys_per_node
```